src/Http/Session/DatabaseSession.php

```diff
@@ -150,14 +149,16 @@ public function write($id, $data): bool
         if (!$id) {
             return false;
         }
-        $expires = time() + $this->_timeout;
-        $record = compact('data', 'expires');
+
         /** @var string $pkField */
         $pkField = $this->_table->getPrimaryKey();
-        $record[$pkField] = $id;
-        $result = $this->_table->save(new Entity($record));
+        $session = $this->_table->newEntity([
+            $pkField => $id,
+            'data' => $data,
+            'expires' => time() + $this->_timeout,
+        ], ['guard' => false]);
```

Comment:
You can't use `guard` option with `newEntity()` either since it's an Entity constructor option.